src/metrics/pocket_area/voronoi_pocket_area.py

```diff
@@ -25,8 +25,7 @@ def voronoi_pocket_area(players: List[Dict]) -> PocketArea:
         # Limit pocket area behind passer. Double the max depth behind passer
         # so that the pocket boundary will fall at the midpoint.
         (passer["x"], passer["y"] - (2 * pocket_max_depth_behind_passer)),
-        # Limit pocket to line of scrimmage (y = 0).
-        # TODO(vinesh): Reorient tracking data so that y = 0 is L.O.S.
```

Comment:
Can you add this TODO back? I haven't implemented that change yet.

Reply:
I just added it again. Sorry for the delay